checks.d/docker.py

```diff
@@ -422,6 +422,9 @@ def _find_cgroup_filename_pattern(self):
                     return os.path.join('%(mountpoint)s/docker/%(id)s/%(file)s')
                 elif os.path.exists(stat_file_path_coreos):
                     return os.path.join('%(mountpoint)s/system.slice/docker-%(id)s.scope/%(file)s')
+                # Kubernetes, we can't test the existence of the path without the id
+                else:
```

Comment:
Indeed, it was a quick fix. :disappointed:  
I updated the PR, passing a `container_id` to `_find_cgroup_filename_pattern` to be able to test the path.